spec.html

```diff
@@ -15268,7 +15272,13 @@ <h1>Static Semantics: Early Errors</h1>
           It is a Syntax Error if AssignmentTargetType of |LeftHandSideExpression| is not ~simple~.
         </li>
       </ul>
-      <emu-grammar>AssignmentExpression : LeftHandSideExpression AssignmentOperator AssignmentExpression</emu-grammar>
+      <emu-grammar>
+        AssignmentExpression :
+          LeftHandSideExpression AssignmentOperator AssignmentExpression
+          LeftHandSideExpression `&amp;&amp;=` AssignmentExpression
+          LeftHandSideExpression `||=`  AssignmentExpression
+          LeftHandSideExpression `??=`   AssignmentExpression
```

Comment:
There's some extra whitespace on this line and the previous. (Someday the linter will warn for this, but it does not yet know how.)

Reply:
Fixed.